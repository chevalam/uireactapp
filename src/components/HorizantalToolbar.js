import React from 'react';
import IconButton from 'material-ui/IconButton';
import ViewModule from 'material-ui-icons/ViewModule';
import Sort from 'material-ui-icons/Sort';
import Settings from 'material-ui-icons/Settings';
import MoreVert from 'material-ui-icons/MoreVert';

const HorizantalToolbar = () =>
        <div style={{justifyContent: 'space-between'}}>
            <IconButton aria-label="View">
                <ViewModule />
            </IconButton>
            <IconButton aria-label="Sort">
                <Sort />
            </IconButton>
            <IconButton aria-label="Settings">
                <Settings />
            </IconButton>
            <IconButton aria-label="More">
                <MoreVert />
            </IconButton>
        </div>

export default HorizantalToolbar;
