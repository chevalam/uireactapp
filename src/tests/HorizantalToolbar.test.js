import React from 'react';
import { shallow, configure } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';
import HorizantalToolbar from '../components/HorizantalToolbar';

const classes = {
    toolbar: '[object Object]'
}
configure({ adapter: new Adapter() });

describe('HorizantalToolbar', () => {
    it('should render correctly', () => {
        const output = shallow(
            <HorizantalToolbar />
        );
        expect(shallowToJson(output)).toMatchSnapshot();
    });
});
