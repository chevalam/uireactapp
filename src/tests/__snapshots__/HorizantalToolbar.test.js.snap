// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HorizantalToolbar should render correctly 1`] = `
<div
  style={
    Object {
      "justifyContent": "space-between",
    }
  }
>
  <IconButton
    aria-label="View"
    disableTouchRipple={false}
    disabled={false}
    iconStyle={Object {}}
    tooltipPosition="bottom-center"
    touch={false}
  >
    <pure(ViewModule) />
  </IconButton>
  <IconButton
    aria-label="Sort"
    disableTouchRipple={false}
    disabled={false}
    iconStyle={Object {}}
    tooltipPosition="bottom-center"
    touch={false}
  >
    <pure(Sort) />
  </IconButton>
  <IconButton
    aria-label="Settings"
    disableTouchRipple={false}
    disabled={false}
    iconStyle={Object {}}
    tooltipPosition="bottom-center"
    touch={false}
  >
    <pure(Settings) />
  </IconButton>
  <IconButton
    aria-label="More"
    disableTouchRipple={false}
    disabled={false}
    iconStyle={Object {}}
    tooltipPosition="bottom-center"
    touch={false}
  >
    <pure(MoreVert) />
  </IconButton>
</div>
`;
